proposed/http-middleware/middleware-meta.md

```diff
@@ -123,11 +120,10 @@ Middleware taking this approach generally has the following commonalities:
   the request for processing.
 * The middleware is passed 2 arguments during invocation:
   1. An object that represents an HTTP request.
-  2. A delegate that receives the request to dispatch next middleware in the pipeline.
+  2. A delegate that receives the request to dispatch next middleware.
```

Comment:
Perhaps this could be: "A delegate by which the middleware may process the request if unable to complete the request alone, or unable to generate a response."

That would get rid of the "next middleware" and "pipeline" and "stack" and whatnot verbiage entirely. We could indicate that delegates often act as queues or stacks, but that these are specific implementation details of specific libraries, and not an architectural consideration for this specification.